packages/swapper/src/swappers/thorchain/types.ts

```diff
@@ -71,3 +71,9 @@ export interface CosmosThorTrade<C extends ChainId> extends Trade<C> {
 }
 
 export type ThorTrade<C extends ChainId> = BtcThorTrade<C> | EthThorTrade<C> | CosmosThorTrade<C>
+
+export type ThorChainUtxoChainIds =
```

Comment:
Wondering if this should just be the definition of `UtxoSupportedChainIds` instead. Then the thorswapper could just use that value in a similar manner as the other swappers.

Reply:
idk I personally like having it twice its more clear to me.